c2corg_api/views/forum.py

```diff
@@ -86,12 +97,25 @@ def __init__(self, request):
     def collection_post(self):
         settings = self.request.registry.settings
 
+        import pprint
         locale = self.request.validated['locale']
+        document = self.request.validated['document']
+        log.warning(pprint.pformat(dir(locale)))
+        document_type = association_keys_for_types[document.type]
+
+        referer = self.request.referer or '/'
+        referer_path = parse_url(referer).path
+        # only keep the left part of referer path
+        referer_scheme_host = referer.rsplit(referer_path, 1)[0]
```

Comment:
if `self.request.referer` is missing, then `referer_scheme_host` will be an empty string.

As the host site will always be www.camtocamp.org, I think we should rather keep it very simple: 

``` python
content = '<a href="https://www.camptocamp.org{}">{}</a>'.format(referer_path)
``` 

And IMO,  rename `referer_path` to `document_path`

Reply:
I applied your suggestion in 7bf9533